clients/admin-ui/src/features/privacy-experience/ConfigurePrivacyExperience.tsx

```diff
@@ -160,12 +185,14 @@ const ConfigurePrivacyExperience = ({
           {translationToEdit ? (
             <PrivacyExperienceTranslationForm
               translation={translationToEdit}
-              onReturnToMainForm={() => setTranslationToEdit(undefined)}
+              isOOB={translationIsOOB}
```

Comment:
note: if you include the "isOOB" state in the `translation` object, this'll be easier to manage 😄